1 ) public class ArithmeticExceptionExample {
    public static void main(String[] args) {
        try {
            int num1 = 10;
            int num2 = 0;
            int result = num1 / num2;
            System.out.println("Result: " + result);
        } catch (ArithmeticException e) {
            System.out.println("Error: Cannot divide by zero!");
        }
    }
} 
out put : Error: Cannot divide by zero.


2 )    public class MultipleExceptionsExample {
    public static void main(String[] args) {
        try {
            String str = null;
            System.out.println(str.length());
            int num = Integer.parseInt("abc");
        } catch (NullPointerException e) {
            System.out.println("Error: Null pointer exception!");
        } catch (NumberFormatException e) {
            System.out.println("Error: Invalid number format!");
        }
    }
}
out put ; Error: Null pointer exception!

3 )    public class CustomExceptionExample {
    public static void main(String[] args) {
        try {
            validateAge(15);
        } catch (AgeException e) {
            System.out.println("Error: " + e.getMessage());
        }
    }

    public static void validateAge(int age) throws AgeException {
        if (age < 18) {
            throw new AgeException("You must be at least 18 years old.");
        }
    }
}

class AgeException extends Exception {
    public AgeException(String message) {
        super(message);
    }
}
out put : Error: You must be at least 18 years old.
